modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/configuration/StorageProfileConfigurationSchema.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.cluster.management.configuration;
+
+import org.apache.ignite.configuration.annotation.Config;
+import org.apache.ignite.configuration.annotation.InjectedName;
+import org.apache.ignite.configuration.annotation.Value;
+
+/**
+ * TODO: add java doc https://issues.apache.org/jira/browse/IGNITE-20564.
+ */
+@Config
+public class StorageProfileConfigurationSchema {
```

Comment:
I would prefer to use the current *StorageEngineConfigurationSchema roots + *DataRegionConfigurationSchema pair instead of introducing this the the next one configurations. Because my current issue 20564 is on pause, we can live with this meaningless stubs in configuration some period of time :(

Reply:
As far as I understand, we will create separate schema for storage profiles. Why do we need then do double work using `StorageEngineConfigurationSchema`  and then implementing again separate schema?

Reply:
I hope that your current issue 20564  won't be on stop a long period of time, so I don't see a big problem in this stub being in the code base, taking into account that you will reuse it soon

Reply:
Ok, I see